describe.py

```diff
@@ -503,6 +520,7 @@ def generate_all_api_documents(
             documentation should be saved.
         artifact_destination_dir (str): relative path where the discovery
             artifacts should be saved.
+        discovery_uri_template (str): URI template of the API's discovery document.
```

Comment:
Same comment here.

Reply:
Fixed in https://github.com/googleapis/google-api-python-client/commit/3863180415eea21b2d74d4e58e65ccb83ed82cae